commands/remove.rb

```diff
@@ -23,28 +23,30 @@ def self.remove(pkg, verbose)
     pkg.preremove
 
     # Remove the files and directories installed by the package.
-    Dir.chdir CREW_CONFIG_PATH do
-      # Remove all files installed by the package.
-      File.foreach(File.join(CREW_META_PATH, "#{pkg.name}.filelist"), chomp: true) do |line|
-        next unless line.start_with?(CREW_PREFIX)
-        if system("grep --exclude #{pkg.name}.filelist -Fxq '#{line}' ./meta/*.filelist")
-          puts "#{line} is in another package. It will not be removed during the removal of #{pkg.name}.".orange
-        else
-          puts "Removing file #{line}".yellow if verbose
-          FileUtils.remove_file line, exception: false
+    unless pkg.is_fake
```

Comment:
This should be `pkg.is_fake?`-- `pkg.is_fake` always returns true so installs are broken right now.

Reply:
See #9770.